packages/nightingale-manager/tests/nightingale-manager.spec.ts

```diff
@@ -0,0 +1,62 @@
+import NightingaleElement from "@nightingale-elements/nightingale-new-core";
+import NightingaleManager from "../src/index";
+
+let rendered: NightingaleManager;
+
+describe("nightingale-navigation tests", () => {
+  beforeEach(() => {
+    rendered = new NightingaleManager();
+    document.documentElement.appendChild(rendered);
+  });
+
+  afterEach(() => {
+    document.documentElement.removeChild(rendered);
+  });
+
+  test("It should propagate reflected attributes", () => {
```

Comment:
I think a better test would be to add 2 `innerElements` and make sure the atrributes get propagated to each other. Just to make sure is not just Lit doing it internally